xrtpy/response/xrt_teem.py

```diff
@@ -0,0 +1,394 @@
+import sys
+import numpy as np
+from astropy.constants import c, h
+from astropy import units as u
+from xrtpy.response.temperature_response import TemperatureResponseFundamental
+from sunpy.coordinates.sun import angular_radius
+
+
+def xrt_teem(hdr1, data1, hdr2, data2, binfac=1, Trange=None,
+             no_threshold=False, Te_err_threshold=0.5,
+             photon_noise_threshold=0.2, mask=None, verbose=False):
+    """
+    Get coronal temperatures and emission measures from a pair of images using
+    the filter ratio method.
+
+    Currently this program uses the solar spectrum calculated with CHIANTI
+    database ver. 6.0.1 (density: 10^9 [cm^-3], ionization equilibrium:
+    chianti.ioneq, abundance: sun_coronal_ext), because this is the only
+    spectrum available in xrtpy. We expect this to change.
+
+    Parameters:
+    -----------
+    hdr1 : fits header
+        header for the first image
```

Comment:
I think that could be a lot to put in a docstring. Do you think someone who doesn't know what a fits header is would want to use this code? Since all the XRT data is in fits files, I think a prerequisite for using it would be to learn about using them.